src/resources/Activities/Activities.ts

```diff
@@ -21,15 +21,20 @@ export default class Activity extends Resource {
     }
 
     list(params?: ListActivitiesParams, activityFacet?: ActivityListingFilters) {
+        const isPublic = activityFacet.sections?.includes('INTERNAL');
```

Comment:
Your conditions are reversed, if it includes INTERNAL it is not public

Reply:
oops you are right, I just changed it should be ok now